rpxdock/app/options.py

```diff
@@ -248,51 +248,70 @@ def default_cli_parser(parent=None, **kw):
       "--ori_resl", default=30.0, type=float,
       help='resolution of top level orientation, sometimes ignored, and resl is taken from hscore data instead. default 30'
    )
-   addarg("--grid_resolution_cart_angstroms", type=float, default=1)
-   addarg("--grid_resolution_ori_degrees", type=float, default=1)
+   addarg("--grid_resolution_cart_angstroms", type=float, default=1, 
+   	  help='cartesian resolution in Angstroms during grid search. default 1')
+   addarg("--grid_resolution_ori_degrees", type=float, default=1, 
+   	  help='rotation orientation resolution in degrees during grid search. default 1')
    # tcdock
    addarg(
       "--architecture", type=str, default=None,
-      help='architecture to be produced by docking. Can be cage I32, O43, T32 or Cx for cyclic. For plug protocol, can be PLUG_Cx. No default value'
+      help='architecture to be produced by docking. Can be cage I32, O43, T32 where larger axis of symmetry is listed first, or Cx for cyclic. For plug protocol, can be PLUG_Cx. No default value'
```

Comment:
We should also add an example of how the dihedrals are called, it's a classic question.